otaku_back/otaku_back/database/base_repository.py

```diff
@@ -0,0 +1,36 @@
+from typing import Generic, TypeVar, List, Optional
+from django.db import models
+
+T = TypeVar('T', bound=models.Model)
+
+class Repository(Generic[T]):
```

Comment:
Make it without using typing module
https://docs.python.org/3/whatsnew/3.12.html#whatsnew312-pep695